packages/vector_graphics_compiler/lib/src/svg/parser.dart

```diff
@@ -540,6 +541,9 @@ class SvgParser {
   final bool _warningsAsErrors;
   final _Resolver _definitions = _Resolver();
   final Queue<_SvgGroupTuple> _parentDrawables = ListQueue<_SvgGroupTuple>(10);
+
+  /// Toggles whether MaskinOptimizer is enabled or disabled.
+  bool enableMaskingOptimizer = true;
```

Comment:
final.

In the doc comment on this, make `[MaskingOptimizer]` a link by putting it in square brackets (and fix the typo :)

Reply:
fixed.